bin/ch-test

```diff
@@ -662,6 +662,21 @@ test_one_file () {
     bats "$one_file"
 }
 
+test_rootemu () {
+    if command -v ch-image > /dev/null 2>&1; then
+        if [[ ("$scope" != "full") && (-z "$with_rootemu") ]]; then
+            echo "skipping root emulation tests (full scope only)"
+        else
+            echo 'executing root emulation tests...'
+            bats force-auto.bats
+            echo yes > "$TMP_/rootemu"
+        fi
```

Comment:
Let's talk if you disagree but I think this conditional should go elsewhere. Then you wouldn't need `--with-rootemu`.